packages/compiler/core/parser.ts

```diff
@@ -2669,13 +2669,18 @@ function visitEach<T>(cb: NodeCallback<T>, nodes: readonly Node[] | undefined):
   return;
 }
 
+export interface GetNodeAtPositionOptions {
+  filter?: (node: Node) => boolean;
+  resolveExact?: boolean;
```

Comment:
This is worth some documentation, but then again, I'm reluctant to bake this into the public docs.

Maybe a better thing would be to make this core function simple and always exact and have a private wrapper helper in language server that adjusts position and calls again if necessary.

Reply:
Yeah I like that